bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetEnergyHandler.java

```diff
@@ -210,4 +253,39 @@ private void updateActivePower(BaseOpenMessage msg) {
             updateState(CHANNEL_POWER, UnDefType.UNDEF);
         }
     }
+
+    /**
+     * Updates current day totalizer
+     *
+     * @param msg the EnergyManagement message received
+     * @throws FrameException
```

Comment:
To be removed, FrameException is not thrown but caught

Reply:
ok